From 33556da9e2ac712b6ae4f890111c4886b4091f91 Mon Sep 17 00:00:00 2001
From: ghsr <ghsr92@yandex.ru>
Date: Fri, 16 Sep 2016 01:14:43 +0600
Subject: [PATCH] charger: add a check WITH_CM_CHARGER flag

This will fix the build error when WITH_CM_CHARGER flag is false and
product AAPT config are not set.

Change-Id: I68a84c39d355f16d8577cf0e4a12589cf8df643c
---
 charger/Android.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/charger/Android.mk b/charger/Android.mk
index f2bdde2..8df755a 100644
--- a/charger/Android.mk
+++ b/charger/Android.mk
@@ -1,5 +1,7 @@
 LOCAL_PATH := $(call my-dir)
 
+ifneq ($(WITH_CM_CHARGER),false)
+
 # Set healthd_density to the density bucket of the device.
 healthd_density := unknown
 ifneq (,$(TARGET_RECOVERY_DENSITY))
@@ -68,3 +70,5 @@ include $(BUILD_PHONY_PACKAGE)
 
 _add-charger-image :=
 _img_modules :=
+
+endif
-- 
2.9.3

